<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Gotevagyok</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="addTextBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>640</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>400</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="storedTextBox">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>580</width>
      <height>640</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>670</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>670</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>331</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Light</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Light</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>date</string>
    </property>
   </widget>
   <widget class="QLabel" name="hrsLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Light</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>hrs</string>
    </property>
   </widget>
   <widget class="QLabel" name="minsLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>80</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Light</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mins</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>175</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerButton">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>430</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usrNameTextBox">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>65</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwTextBox">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>115</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>75</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>125</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>password</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>0</y>
      <width>20</width>
      <height>720</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="regUsrNameTextBox">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>270</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="regPasswTextBox">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>320</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="regPasswTextBoxCheck">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>370</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>330</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>280</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>380</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>password</string>
    </property>
   </widget>
   <widget class="QLabel" name="profilePic">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>profilePicture.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="timestampCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Add timestamps</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>500</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>options</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>219</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
